test/checks/aria/unsupportedrole.js

```diff
@@ -12,7 +12,7 @@ describe('unsupportedrole', function() {
 			type: 'widget',
 			unsupported: true
 		};
-		fixture.innerHTML = '<div id="target" role="mccheddarton">Contents</div>';
+		fixture.innerHTML = '<div id="target" role="mcheddarton">Contents</div>';
```

Comment:
Not a big deal at all, but what's a "mcheddarton"?

Reply:
I believe it's the name of Marcy's dog.

Reply:
Haha, ok